client/ionic/src/pages/About.tsx

```diff
@@ -47,32 +47,85 @@ function useLicenseTexts() {
   return licenses;
 }
 
+// TODO: DRY with above, funtional per content type / relative path
+// TODO: INTL
+
+function usePrivacyPolicy() {
+  const [privacyPolicy, setPrivacyPolicy] = useState(null as string | null);
+
+  useEffect(() => {
+    async function fetchPrivacyPolicy() {
+      const url = `${process.env.PUBLIC_URL}/assets/content/privacy-policy.txt`;
```

Comment:
This is fine since this is how we're fetching content in the rest of v0.1 -- but @advayDev1 have we thought about how we would make this code work fine offline? Presumably we'd need to fallback to an offline file if the fetch fails.

Reply:
Not sure I understand.  This is a fetch of a local file.

Reply:
Please resolve this post this PR.

Reply:
Oh, I assumed PUBLIC_URL pointed to a http url -- so it points to something local?